frontend/src/employee-frontend/components/voucher-value-decision/VoucherValueDecisionSummary.tsx

```diff
@@ -38,7 +38,7 @@ export default React.memo(function Summary({ decision }: Props) {
         <TotalTitle noMargin>
           {i18n.valueDecision.summary.totalValue}
         </TotalTitle>
-        <b>{formatCents(decision.voucherValue - decision.coPayment)} €</b>
+        <b>{formatCents(decision.voucherValue - decision.finalCoPayment)} €</b>
```

Comment:
🤔 was this intentional?

Reply:
This should be a part of another pull request 🙈

Reply:
Fixed 👍